CONTRIBUTING.md

```diff
@@ -22,11 +22,12 @@ To contribute to this project, it is recommended that you follow these steps:
 4. Run the unit tests on your system (see Testing section). At this point, if any tests fail, do not
 begin development. Try to investigate these failures. If you're unable to do so, report an issue
 through our [GitHub issues page](http://github.com/ni/nidaqmx-python/issues).
-5. Write new tests that demonstrate your bug or feature. Ensure that these new tests fail.
-6. Make your change.
-7. Run all the unit tests again (which include the tests you just added), and confirm that they all
+5. Once the necessary changes are done, update the auto-generated code using ``poetry run python -m codegen --dest generated/nidaqmx``. This will ensure that the latest files are present in ``generated`` folder.
```

Comment:
I think this should be after the "Make your change." step?

Reply:
Or put it in a separate section about updating the codegen.